llvm/test/Transforms/SimplifyCFG/X86/hoist-loads-stores-with-cf.ll

```diff
@@ -674,6 +672,62 @@ if.false:
   ret void
 }
 
+define i32 @str_transcode0(i1 %cond1, ptr %p, i1 %cond2) {
```

Comment:
Probably rename this function? I assume the function name should show what it's testing.

Reply:
https://github.com/llvm/llvm-project/pull/109638